config/boards/radxa-zero3.wip

```diff
@@ -0,0 +1,36 @@
+# Rockchip RK3566 quad core
+BOARD_NAME="Radxa ZERO 3"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER="Radxa"
+BOOTCONFIG="radxa-zero3-rk3566_defconfig"
+KERNEL_TARGET="legacy"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3566-radxa-zero3.dtb"
+IMAGE_PARTITION_TABLE="gpt"
+BOOT_SCENARIO="spl-blobs"
+BOOT_SUPPORT_SPI="yes"
+BOOTFS_TYPE="fat"
+
+# Override family config for this board; let's avoid conditionals in family config.
+function post_family_config__radxa-zero3_use_vendor_uboot() {
+	BOOTSOURCE='https://github.com/radxa/u-boot.git'
+	BOOTBRANCH='branch:rk35xx-2024.01'
+	BOOTPATCHDIR="u-boot-radxa-latest"
+
+	UBOOT_TARGET_MAP="BL31=$RKBIN_DIR/$BL31_BLOB ROCKCHIP_TPL=$RKBIN_DIR/$DDR_BLOB;;u-boot-rockchip.bin u-boot-rockchip-spi.bin"
+
+	unset uboot_custom_postprocess write_uboot_platform write_uboot_platform_mtd
+
+	function write_uboot_platform() {
+		dd if=$1/u-boot-rockchip.bin of=$2 seek=64 conv=notrunc status=none
+	}
+
+	function write_uboot_platform_mtd() {
+		flashcp -v -p u-boot-rockchip-spi.bin /dev/mtd0
```

Comment:
I missed this initially; you need `"$1/u-boot-rockchip-spi.bin"` here for this to work

Reply:
I remembered there is no SPI flash on ZERO 3E/3W ;)

Reply:
@rpardini since you approved I assume this is outdated/resolved?

Reply:
Yeah! There's no SPI on the board..